collective/zopeedit/zopeedit.py

```diff
@@ -273,6 +273,7 @@ def __init__(self, input_file=""):
                 self.metadata["content_type"] = self.dexterity.get(
                     "Content-Type", self.metadata.get("content_type", "text/plain")
                 )
+                in_f.close()  # to avoid problem when deleting original zem file
```

Comment:
Weird :) 
It seems to me the code uses `in_f` just to read it.
You can try to miminize the time the file is open by using a context manager right before the file is used, e.g.:
```
with open(self.input_file, "rb") as f:
      msg = email.message_from_string(f.read())
```

Just my 2c.

Reply:
@ale-rt as the file is previously opened, maybe replaced following a condition and used later, it is not suitable here to use a with statement.